docker/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM frolvlad/alpine-python3
+
+RUN apk update && apk add --no-cache \
+   git \
+   python3-dev \ 
+   libffi \
+   libffi-dev \
+   musl-dev \
+   gcc \
+   openssl-dev \
+   g++
+
+
+RUN git clone https://github.com/geopython/pygeoapi.git
+
+ENV PYGEOAPI_CONFIG=/pygeoapi/local.yml
+
+WORKDIR /pygeoapi
+RUN pip3 install -r requirements.txt
+RUN pip3 install -r requirements-dev.txt
+RUN pip3 install -e .
+RUN cp pygeoapi-config.yml local.yml
+#export PYGEOAPI_CONFIG=`pwd`/local.yml
+
```

Comment:
I think we also have to generate the OpenAPI and set `PYGEOAPI_OPENAPI`, i.e.:

```docker
RUN pygeoapi generate_openapi_document -c $PYGEOAPI_CONFIG > /pygeoapi/openapi.yml
ENV PYGEOAPI_OPENAPI=/pygeoapi/openapi.yml
```

Reply:
The dockerfile was not supposed to be in this PR will remove it